src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/AppDesigner.xaml

```diff
@@ -27,7 +26,6 @@
                   Visible="false">
     <StringProperty.DataSource>
       <DataSource HasConfigurationCondition="false"
-                  Label="Configuration"
```

Comment:
>Label [](start = 18, length = 5)

I'm guessing this is cleanup/orthogonal to the fix?

Reply:
It's related, as we started writing this property, we would write out a special ItemGroup with a Label="Confioguration". We never hit it because we weren't writing it out.